google/ggrc-core
# GGRC-813 Fix LHN tabs width on panel resizing

# Issue description

LHN tabs don't change width when user resize panel.

# Steps to test the changes

1. Open LHN on My Work page
2. Change the size (e.g. extend right) and back

**Actual Result:** LHN tabs width is not the same as LHN panel
**Expected Result:** LHN tabs should have the same width as LHN panel.

# Solution description

Set tabs width using css styles instead of js.
Also set LHN height in css.
Remove unneeded js code and css classes.

# Sanity checklist




# PR Review checklist
